packages/terra-notification-dialog/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Added
+  * Added FocusLock to confine focus within the modal popup.
```

Comment:
this is not correct !! we need to add logs about the changes made for notification-dialog which has got JEST SNAPSHOT update

Reply:
updated.